### PR TITLE
ocp 13065 upgrade auto

### PR DESCRIPTION
**Pull request by sunilcio**
ocp 13065 upgrade auto

### CODE REVIEW

**Comment by sunilcio** [comment]
@liangxia  @pruan-rht PTAL
I was updating my last PR (#1166) but got mixed up with other vm. Created this one with your suggested changes.
Case runs correctly with the changes http://pastebin.test.redhat.com/873865

File: features/upgrade/node/scc.feature [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+  # @author sunilc@redhat.com
+  @upgrade-prepare
+  @admin
+  Scenario: Upgrade - Seccomp part of SCC policy should be kept and working after upgrade - prepare
+    Given I switch to cluster admin pseudo user
+    When I run the :new_project client command with:
+      | project_name | node-upgrade-scc |
+    And I use the "node-upgrade-scc" project
+    When I run the :create client command with:
```
Comment (pruan-rht) [inline features/upgrade/node/scc.feature:11]: @sunilcio please use the step `Given I obtain...` to retrieve the testdata.  Please refer #1227 as example of how to do it.

Comment (jhou1) [reply features/upgrade/node/scc.feature:11]:
```
Given I obtain test data file "testdata/node/scc.yaml"
When I run the :create client command with:
  | f | testdata/node/scc.yaml |
```

File: features/upgrade/node/scc.feature [hunk 2]
```diff
@@ -0,0 +1,22 @@
+Feature: Seccomp part of SCC policy should be kept and working after upgrade	
+
+  # @author sunilc@redhat.com
+  @upgrade-prepare
+  @admin
+  Scenario: Upgrade - Seccomp part of SCC policy should be kept and working after upgrade - prepare
+    Given I switch to cluster admin pseudo user
+    When I run the :new_project client command with:
+      | project_name | node-upgrade-scc |
+    And I use the "node-upgrade-scc" project
```
Comment (liangxia) [inline features/upgrade/node/scc.feature:10]: SCC is a cluster resource. I don't think we need the project here as it's not used actually.

File: features/upgrade/node/scc.feature (hunk 1 above)
Comment (liangxia) [reply features/upgrade/node/scc.feature:11]: By the way, creating scc should be destructive, as scc can significantly affect a lot of scenarios.

**Pull request closed without merging**